fwoa-tools/src/migrationImport.ts

```diff
@@ -342,7 +333,19 @@ async function deleteResourcesInBundle(deletePaths: string[]): Promise<void> {
       writeFileSync(`${IMPORT_STATE_FILE_NAME}`, JSON.stringify(successfullyCompletedFolders));
     }
   }
-  logs.end();
+}
+
+/* istanbul ignore next */
+(async () => {
+  // don't runScript when importing code for unit tests
+  if (!process.argv.includes('test')) {
```

Comment:
We switched to using `if (!process.env.UNIT_TEST) {` since we now pass `UNIT_TEST` as an environment variable when running `rushx test`. Before when we ran `rushx test` we save that it wasn't passing `test` as an argument.